src/Compilers/CSharp/Portable/Symbols/ArrayTypeSymbol.cs

```diff
@@ -249,6 +249,14 @@ internal sealed override bool IsByRefLikeType
             }
         }
 
+        internal sealed override bool IsReadOnly
```

Comment:
We should probably add tests or update existing tests to verify this on various kinds of symbols.